libraries/libbls/libbls.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/hex"
+	"log"
+
+	"github.com/ElrondNetwork/elrond-go-crypto/signing"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl"
+	"github.com/ElrondNetwork/elrond-go-crypto/signing/mcl/singlesig"
+)
+
+var (
+	keyGenerator = signing.NewKeyGenerator(mcl.NewSuiteBLS12())
+	blsSigner    = singlesig.BlsSingleSigner{}
+)
+
+func doGeneratePrivateKey() string {
+	privateKey, _ := keyGenerator.GeneratePair()
+	privateKeyBytes, err := privateKey.ToByteArray()
+	if err != nil {
+		log.Println("doGeneratePrivateKey(): error when decoding the private key", err)
```

Comment:
Chosen this pattern: log & return empty result, instead of returning a struct holding the payload and an error code (plus an error message) - easier on lib's side, easier on consumer's side. 

All right as it is, or should we switch to structs? E.g. https://github.com/golang/go/issues/18412#issuecomment-268847417